CHANGELOG.txt

```diff
@@ -3,6 +3,13 @@ Changes
 
 0.12.4, XX/XX/XXXX
 
+* Word2vec default values of hyperparameters now are identical to those of original C tool, generally improving performance (Andrey Kutuzov, #538)
+ - By default use CBOW with negative sampling and the mean of context vectors, instead of skip-gram with hierarchical softmax.
```

Comment:
Unnecessarily verbose -- a github link to the diff between "old vs. new" defaults is enough IMO.